src/runtime/virtcontainers/device/drivers/block.go

```diff
@@ -18,6 +18,11 @@ import (
 
 const maxDevIDSize = 31
 
+const (
+	FsTypeOpt      string = "fstype"
```

Comment:
Non-package public const need not be started with uppercase letters.

Reply:
@liubin thanks, already commit again.